### PR TITLE
RDM-3381: Display logged-in user and "Sign out" link

### PR DESCRIPTION
**Pull request by Daniel-Lam**
### JIRA link ###
https://tools.hmcts.net/jira/browse/RDM-3381.

### Change description ###
Show the current user's forename and surname on the right-hand side of the header, along with a link to sign out of the application.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/routes/confirmDeleteItem.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ router.get("/deleteitem", (req, res, next) => {
   if (req.adminWebAuthorization && req.adminWebAuthorization.canManageDefinition) {
     const responseContent: { [k: string]: any } = {};
     responseContent.adminWebAuthorization = req.adminWebAuthorization;
+    responseContent.user = JSON.stringify(req.authentication.user);
```
Comment (gbenadikar) [inline src/main/routes/confirmDeleteItem.ts]: can this be unit tested?

File: src/main/routes/jurisdiction.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ router.get("/jurisdiction", (req, res, next) => {
     res.status(200);
     const responseContent: { [k: string]: any } = {};
     responseContent.adminWebAuthorization = req.adminWebAuthorization;
+    responseContent.user = JSON.stringify(req.authentication.user);
```
Comment (gbenadikar) [inline src/main/routes/jurisdiction.ts]: can this be unit tested?

File: src/main/routes/confirmDeleteItem.ts (hunk 1 above)
Comment (Daniel-Lam) [reply src/main/routes/confirmDeleteItem.ts]: @gbenadikar - I've amended the tests to check for the presence of the user data within a hidden `input` on the HTML page

File: src/main/routes/jurisdiction.ts (hunk 2 above)
Comment (Daniel-Lam) [reply src/main/routes/jurisdiction.ts]: @gbenadikar - I've amended the tests to check for the presence of the user data within a hidden `input` on the HTML page

**Pull request merged**